synapse/models/inet.py

```diff
@@ -1927,6 +2078,9 @@ def getModelDefs(self):
                         ('host', ('it:host', {}), {
                             'doc': 'The host that used the network egress.'}),
 
+                        ('account', ('inet:service:account', {}), {
```

Comment:
Oops one  more:
1. - :account seems out of place? An account using an egress implies use in time - either a one-time observation or "used X egress between Y and Z". Is someone expected to use "seen" to bound a specific combo of host/address/account? Time / duration seems like it should be a bit more formal in this case. (Multiple users/accounts can use a single egress - e.g., corporate egress IP, TOR exit node).
2. inet:egress seems very "standalone" except for the associated IP. How does / should an egress relate (link) to things like inet:tunnel, inet:flow, etc.? This seems like a gap?
3.  In other network-related forms where we have :client and / or :server we have broken out :client:ip(v4|v6), :client:port, and sometimes :client:proto. Should we do that here?